<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketWidget</class>
 <widget class="QWidget" name="TicketWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>797</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_CaseID">
        <property name="text">
         <string>CaseID</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_CaseID</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_CaseID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_DateOpened">
        <property name="text">
         <string>Date Opened</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_DateOpened</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_DateOpened">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_DateClosed">
        <property name="text">
         <string>Date Closed</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_DateClosed</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_DateClosed">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_Hostopian">
        <property name="text">
         <string>Hostopian</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Hostopian</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_Hostopian"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_Category">
        <property name="text">
         <string>Category</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Category</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_Category"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Domain">
        <property name="text">
         <string>Domain</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_Domain</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_Domain"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_SubCategory">
        <property name="text">
         <string>Sub Category</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_SubCategory</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="comboBox_SubCategory"/>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_CustName">
        <property name="text">
         <string>Customer Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_CustName</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_CustName"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_Priority">
        <property name="text">
         <string>Priority</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Priority</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QComboBox" name="comboBox_Priority">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_Email">
        <property name="text">
         <string>Email</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_Email</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_Email"/>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_Status">
        <property name="text">
         <string>Status</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Status</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QComboBox" name="comboBox_Status"/>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_OpenedBy">
        <property name="text">
         <string>Opened by</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_OpenedBy</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_OpenedBy">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_Group">
        <property name="text">
         <string>Assigned to Group</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Group</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QComboBox" name="comboBox_Group"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_ClosedBy">
        <property name="text">
         <string>Closed by</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_ClosedBy</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_ClosedBy">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_Person">
        <property name="text">
         <string>Assigned to Person</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QComboBox" name="comboBox_Person"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Problem</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_Problem"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_Notes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="accessibleName">
       <string/>
      </property>
      <attribute name="title">
       <string>Resolution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_Resolution"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Audit Trail</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_AuditTrail"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_Reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Update">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit_History">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
